app/javascript/local/dropzone-cfg.js

```diff
@@ -36,6 +36,7 @@ DropzoneCfg.prototype.init = function () {
     this.$statusTag.classList.remove("govuk-tag--yellow")
     this.$statusTag.classList.add("govuk-tag--green")
     this.$statusTag.innerHTML = "Uploaded"
+    location.reload()
```

Comment:
Quick and dirty solution to get e2e up. The uploaded file was displayed via the js adding it to the page. The delete button is currently only in the html, and would need a bit of wrangling to create in the js

Reply:
I tackled this in the branch I created for sprint demo, if you want to use it? https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/commit/87d2fb6ca2af7cfcb40fd9849dc48fc59cea155f

Reply:
Ooh, helpful! Thank you!